### PR TITLE
Fix PED bugs

### LINKED ISSUE #126: [PE-D][Tester B] Title screen has unknown characters shown on the Welcome screen
## Description
Non-ascii characters displayed in the command line interface

## Screenshot

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: wenenhoe/ped#2

### PR DESCRIPTION
**Pull request by NicholasTanYY**
- unknown avatar chars printed to screen (fixed)
- mark: index out of bounds exception when task index is out of range for days with tasks (fixed)
- mark: null pointer exception for days without any tasks (fixed)
- priority: update UG to include information on where it can be found (already existed, tester might have missed it)
- priority: index out of bounds exception when task index is out of range for days with tasks (fixed)
- priority: null pointer exception for days without any tasks (fixed)
- priority: invalid priority results in it being set to L by default (fixed)
- priority: suddenly changes to M when logged out and logged back in (fixed)

Resolves #126, #129, #130, #134, #135, #136

### CODE REVIEW

File: src/main/java/ui/AvatarUi.java [hunk 1]
```diff
@@ -3,31 +3,27 @@
...
+            "           +#%#    +=   =%%*=           \n" +
+            "    *#**#%%%###   *#%=  =###%#%##**++   \n" +
+            "###%%#######**#    %#   =#**########%#**\n" +
+            "++%####**###**%   =##   *#**###***###%++\n" +
+            "##%###########%#  *##=  #############%##\n" +
+            " *%#%%##########+ #%%* *%#########%##%%#\n" +
+            "+%%%%%%####%%#%%#=%%%#=#%%#%%#*#%%%%%%%=\n" +
+            "*@%%%%@%%%#%%%%%%#%%%%##%%%%##%%%@%%%%% \n" +
+            "#@%%%%@%%%%%%%%%%%%@@%%%%%%%%%%%%@%%%%@=\n";
+
```
Comment (Mahesh1772) [inline src/main/java/ui/AvatarUi.java:26]: Good job on fixing the chars

File: src/main/java/storage/Storage.java [hunk 2]
```diff
@@ -142,9 +142,9 @@ private static void configureStatuses(LocalDate date, String markedStatus, Strin
      * @param recentlyAddedTask Task that was most recently added.
      */
     private static void setPriorityLevelStatus(String priorityLevel, Task recentlyAddedTask) {
-        if (!priorityLevel.equals("H")) {
-            recentlyAddedTask.setPriorityLevel(TaskPriorityLevel.MEDIUM);
-        } else if (!priorityLevel.equals("M")) {
+        if (priorityLevel.equals("H")) {
+            recentlyAddedTask.setPriorityLevel(TaskPriorityLevel.HIGH);
+        } else if (priorityLevel.equals("M")) {
```
Comment (Mahesh1772) [inline src/main/java/storage/Storage.java:147]: Nice job spotting the error in priority assignment!

File: src/main/java/data/SetPriorityException.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package data;
+
+import java.util.List;
```
Comment (Mahesh1772) [inline src/main/java/data/SetPriorityException.java:3]: Great idea to add a new class for handling priority related exceptions

File: src/main/java/data/MarkTaskException.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
+package data;
```
Comment (Mahesh1772) [inline src/main/java/data/MarkTaskException.java:1]: Good job to have a new class for task marking exceptions

**Review by Mahesh1772 — COMMENTED** [review]
LGTM!

File: src/main/java/data/MarkTaskException.java (hunk 4 above)
Comment (clarencepohh) [reply src/main/java/data/MarkTaskException.java:1]: Seconded! Might want to write some test cases for the new class in the coming days.

File: src/main/java/data/SetPriorityException.java
Comment (clarencepohh) [inline src/main/java/data/SetPriorityException.java:1]: Similarly, might want to add exception testing for this class as well? I think it has not been implemented as of now.

File: src/main/java/ui/AvatarUi.java (hunk 1 above)
Comment (clarencepohh) [inline src/main/java/ui/AvatarUi.java:26]: Sad to see the OG CLI-nton go :( but it is a fix nonetheless.
Have you considered reducing sizes further to help increase screen space - especially on smaller screens?

**Review by clarencepohh — COMMENTED** [review]
Thanks for fixing a majority of the PE-D bugs!

**Pull request merged**